packages/zoe/src/contracts/attestation/prerequisites.js

```diff
@@ -0,0 +1,93 @@
+// @ts-check
+
+import { AmountMath } from '@agoric/ertp';
+import { E } from '@agoric/eventual-send';
+
+import { subtractOrMakeEmpty } from './helpers';
+
+const { details: X, quote: q } = assert;
+
+// TODO: Make the attestation contract more generic by allowing this
+// to be parameterized
+
+/**
+ * Assert the cosmos-specific prerequisites
+ *
+ * @param {{getAccountState: (address: Address) => {total: Amount, bonded: Amount, locked: Amount,
+ * currentTime: Timestamp}}} cosmos
```

Comment:
Can we use some other term besides "cosmos"?  Having an object that can report the status of staked tokens is a much smaller API than the entire Cosmos SDK chain, and it is also not inherently a Cosmos-specific API.

Maybe "stakeManager" or "stakeReporter"?  I don't want to bikeshed, so any more specific name you suggest would be fine.

Reply:
`stakeReporter` sounds good to me, and is much better than `cosmos`.